packages/admin-stories/src/docs/comet/admin/savesplitbutton.stories.mdx

```diff
@@ -0,0 +1,173 @@
+import { Meta, Story, Canvas } from "@storybook/addon-docs/blocks";
+
+import { SaveSplitButton } from "@comet/admin";
+import { Button, Typography } from "@material-ui/core";
+
+# Components
+
+<Meta title="Docs/@comet/admin/savesplitbutton" />
+
+## Save / Save and go Back Split Button
+
+<Canvas>
+    <Story name="Save / Save and go back">
+        {() => {
+            const Story = () => {
+                const [saving, setSaving] = React.useState(false);
+                return (
+                    <SaveSplitButton
+                        localStorageKey={"Page1.SaveSplitButton"}
+                        saving={saving}
+                        onSavePressed={() => {
+                            // simulating saving process
+                            setSaving(true);
+                            setTimeout(() => {
+                                setSaving(false);
+                            }, 1000);
+                        }}
+                        onSaveAndGoBackPressed={() => {
+                            // simulating saving process
+                            setSaving(true);
+                            setTimeout(() => {
+                                setSaving(false);
+                            }, 1000);
+                        }}
```

Comment:
have you also considered an API like that?
```
<SaveSplitButton>
    <SaveButton onClick=[...}>
    <SaveAndGoBackButton onClick=[...}>
</SaveSplitButton>
```

Reply:
i made this Mr before the toolbar. I already estimated with a note like that. 

Yeah maybe this should the way it should be. I will give it a try